app/models/user.rb

```diff
@@ -0,0 +1,34 @@
+class User < ApplicationRecord
+  has_secure_password
+  validates :email, presence:true, uniqueness:true
+  validates :username, presence:true, uniqueness:true
+  validates :password, length: {minimum: 8}, allow_blank: false
+  before_create{generate_token(:remember_token)}
+  has_many :events, foreign_key: 'user_id', class_name: 'Event'
```

Comment:
`[Optional]` Guys, if you are using the standard name (in this case `:events`), you don't need to specify the `:foreign_key` and `:class_name`